### ISSUE TITLE
[ui] Allow the user to mark a notification as resolved

### ISSUE THREAD

**Issue by nekron** [instedd/aida#226]
This is a nice to have

For #152
Depends on #225

**Comment by nekron** [comment]
We should discuss what does this imply.

Having a chat window open would suggest that a conversation could be maintained between the operator and the user, without the bot interfering, which currently does not happen.

Even if the user replies using the human override keyword to ensure that the answer reaches the operator, it will appear as a different message question in the UI.

We could have a way to disable the bot temporarily to enable such communication, and we could show all the messages for a given session id as part of the same conversation, but that's a different amount of effort altogether.

Right now, only having this checkbox to mark the conversation as solved, seems to bring more confusion than benefits.

**Comment by nditada** [comment]
I think the extra complications stem from the fact that there's ambiguity about two possible modes of interacting:
1-The human override is just a way to respond to a single request. Once the message has been responded by the operator, the chatbot resumes  it's functioning normally. To make an electric analogy, this would be like a "pulse" switch. In this case the "mark as resolved" makes no sense. 
2-The human override is a way to hand over the conversation to the operator. The operator takes over until they decide to transfer the control back to the chatbot. To continue the electric analogy, this would be like a "toggle" switch. In this case the "mark as resolved" is what would actually have the bot resume the conversation. 

If we are already at (1), let's just keep it consistent. We can always add the (2) behavior later.

**Comment by nekron** [comment]
I've created #249 in order to better accommodate this behavior
